tests/components/zeroconf/conftest.py

```diff
@@ -0,0 +1,18 @@
+"""conftest for zeroconf."""
```

Comment:
If we put it  in the `zeroconf` folder, it will only be loaded for the tests in this folder and subfolders of this folder. If any integration would be to set up zeroconf, it would still hit the original function.

Reply:
> If we put it in the `zeroconf` folder, it will only be loaded for the tests in this folder and subfolders of this folder. If any integration would be to set up zeroconf, it would still hit the original function.

I missed what you were getting add.  I moved it one level up.